### DISCUSSION TITLE
In the velero-policy.json, why do we need that ec2 statement?

### DISCUSSION THREAD

**Discussion by spacenasa** [vmware-tanzu/velero#4915] [Community support Q&A]
In the velero-policy.json below, why do we need that ec2 statement?

cat > velero-policy.json <<EOF
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "ec2:DescribeVolumes",
                "ec2:DescribeSnapshots",
                "ec2:CreateTags",
                "ec2:CreateVolume",
                "ec2:CreateSnapshot",
                "ec2:DeleteSnapshot"
            ],
            "Resource": "*"
        },

**Comment by sseago** [comment]
The volume and snapshot related ones are needed for backing up and restoring ebs volumes via snapshotting. I'm less clear on the CreateTags one.
